### ISSUE TITLE
ContextMenuArea doesn't follow Widget Styles variants

### ISSUE THREAD

**Issue by SubZ69** [slint-ui/slint#9646]
### Bug Description

ContextMenuArea stay on light variant even if dark is selected. 

### Reproducible Code (if applicable)

```slint

```

### Environment Details

- Slint Version: 1.13.1b1
- Platform/OS: Windows 11
- Programming Language: Python
- Backend/Renderer: default

**Comment by ogoffart** [comment]
On windows, we use muda native rendering for popup menu, and indeed, that should follow the platform style and not the slint style.

This was also reported in https://github.com/slint-ui/slint/issues/8092

And one comment suggest a way to fix it.
